### ISSUE TITLE
Tensorflow is missing

### ISSUE THREAD

**Issue by manolo75** [boy1dr/SpleeterGui#104]
Hi,

I tried to install SpleeterGui but when I try to run it on a track, I always get an error concerning Tensorflow 👍 
  File "C:\Users\Manu\AppData\Roaming\SpleeterGUI\python\Lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 64, in <module>
    from tensorflow.python._pywrap_tensorflow_internal import *
ImportError: DLL load failed: Le module spécifié est introuvable.

Regards.

Manu

**Comment by manolo75** [comment]
Forget it. I installed this and now it is working : https://aka.ms/vs/16/release/vc_redist.x64.exe

**Comment by boy1dr** [comment]
Glad you got it working :)
This is common problem # 4 on https://makenweb.com/spleeter_help
Would you have any suggestions on how to reword it to help the next person with troubles?
